devel/ninja/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           github 1.0
 PortGroup           muniversal 1.0
 
 epoch               1
-github.setup        ninja-build ninja 1.8.2 v
+github.setup        ninja-build ninja 1.9.0 v
 revision            1
```

Comment:
The `revision` line needs to be reset to 0 when upgrading the `version`.

Reply:
Oops, I forgot to remove that. Thanks, I've fixed it.